inlang/guides/how-to-inlang-app/how-to-inlang-app.md

```diff
@@ -2,23 +2,54 @@
 
 Inlang is a versatile globalization ecosystem that empowers developers to seamlessly integrate language translation capabilities into their applications. This comprehensive guide will walk you through the process of building a general Inlang app with the `@inlang/sdk`. 
 
-We'll cover initializing a project, creating and updating messages, and working with lint rules.
+We'll cover initializing a project from a repo, creating and updating messages, and working with lint rules.
 
-## 1. Initialize the project
+## 1. Open Repository
 
-The first step is to initialize your project using the Inlang SDK. The `loadProject()` function in initializes an Inlang `object` with various functions.
+All Inlang projects live in a repo, so first we use the lix client to open the repo.
+
+We can use node:fs for repos which are already in the filesytem e.g. in the current working directory:
+
+```typescript
+import fs from "node:fs/promises"
+import { openRepository } from "@lix-js/client"
+
+const repoRoot = process.cwd()
```

Comment:
@jldec the repo root is not `process.cwd()`. use `findRepoRoot()` from lix please

Reply:
Thanks @samuelstroschein - that's helpful feedback. I just pushed another [commit](https://github.com/opral/monorepo/pull/2040/commits/d09f7a2be1c5992df3162b0db6ebc057bc8deb02) - loosely based on the [paraglide compile](https://github.com/opral/monorepo/blob/main/inlang/source-code/paraglide/paraglide-js/src/cli/commands/compile.ts#L23-L44) command. Open to suggested re-wording.

Reply:
@jldec can you form an opinion regarding https://github.com/opral/monorepo/pull/2020#discussion_r1453536347 ? 

We have a problem here. `mockRepo` seems to be confusing.

Reply:
see https://github.com/opral/monorepo/discussions/2061

> - mockRepo() is just for testing, and only supports memory-fs.
> - openRepository() is safe to use with non-git operations (just working tree / fs operations) when there is no .git repo initialized.